### PR TITLE
Delete multiple persons

### PR DESCRIPTION
**Pull request by April0616**
Delete 1, 2, 3

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -26,50 +26,63 @@
...
-            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Parameters: INDEX (must be positive integers)\n"
             + "        OR  " + PREFIX_TAG + "TAG (case-sensitive)\n "
             + "Example: " + COMMAND_WORD + " 1\n"
+            + "         " + COMMAND_WORD + " 1, 2, 3\n"
             + "         " + COMMAND_WORD + " " + PREFIX_TAG + "friend\n"
             + "         " + COMMAND_WORD + " " + PREFIX_TAG + "friend " + PREFIX_TAG + "enemy";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    //need to check here
```
Comment (CindyTsai1) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java]: should this comment be deleted

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 2]
```diff
@@ -26,50 +26,63 @@
...
             }
 
-            ReadOnlyPerson personToDelete = lastShownList.get(targetIndex.getZeroBased());
-
             try {
-                model.deletePerson(personToDelete);
+                model.deletePersons(deletePersonList);
             } catch (PersonNotFoundException pnfe) {
-                assert false : "The target person cannot be missing";
+                assert false : "One of the target persons is missing";
```
Comment (CindyTsai1) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:78]: Wanna specify which one missing?

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 3]
```diff
@@ -26,50 +26,63 @@
...
             }
 
-            return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, personToDelete));
+            return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS));
```
Comment (CindyTsai1) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java]: wanna print out who are deleted?

**Review by CindyTsai1 — COMMENTED** [review]
looks good, suggested some changes

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java (hunk 2 above)
Comment (April0616) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java]: ....I'll send another PR later this week to fix this problem!

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java (hunk 3 above)
Comment (April0616) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java]: ....I'll send another PR later this week to fix this problem!

Comment (April0616) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java]: yes..i want ...

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java (hunk 1 above)
Comment (April0616) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java]: I haven't finished this part ..haha

Comment (CindyTsai1) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java]: hahhah ok

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java (hunk 3 above)
Comment (CindyTsai1) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java]: haha oops sorry i didnt see ongoing haha

**Pull request merged**